plant/plants/requests.py

```diff
@@ -0,0 +1,19 @@
+import environ
+import requests
+
+
+def fetch_plant_families():
+    env = environ.Env()
+    response = requests.get(
+        f"https://trefle.io/api/v1/families?token={env("API_PLANT")}",
```

Comment:
API_PLANT should be moved to settings and then used as
`f"https://trefle.io/api/v1/families?token={settings.API_PLANT}",`

I would also consider add key suffix to the name, because API_PLANT lacks a bit of information (is it url or token)